### PR TITLE
update EtcdKVImpl get&delete request builder

### PR DESCRIPTION
**Pull request by AdoHe**

### CODE REVIEW

File: src/main/java/com/coreos/jetcd/op/Op.java [hunk 1]
```diff
@@ -113,5 +113,4 @@ RequestOp toRequestOp() {
             return RequestOp.newBuilder().setRequestDeleteRange(delete).build();
         }
     }
-
-}
+}
```
Comment (xiang90) [inline src/main/java/com/coreos/jetcd/op/Op.java]: add \n?

**Comment by xiang90** [comment]
lgtm

File: src/main/java/com/coreos/jetcd/op/Op.java (hunk 1 above)
Comment (AdoHe) [inline src/main/java/com/coreos/jetcd/op/Op.java]: just remove the extra "\n".

**Comment by AdoHe** [comment]
@xiang90 I am considering whether we should re-use `PutOp` etc in `EtcdKVImpl.java` , now there are some duplicated code in `Op.java` and `EtcdKVImpl.java`.

**Pull request merged**